app/helper/helper.go

```diff
@@ -31,3 +33,21 @@ func ToBigInt(value string) (*big.Int, error) {
 
 	return amount, nil
 }
+
+// ToMirrorNodeTransactionID parses TX with format `0.0.X@{seconds}.{nanos}` to format `0.0.X-{seconds}-{nanos}`
+func ToMirrorNodeTransactionID(txId string) string {
```

Comment:
- what do you think about the following approach when it comes to helper's:
https://play.golang.org/p/Ey07k5HGSGV
I think that helper is becoming the go to file when we do not see a proper place to fit the functionality and this would evolve into a code that it is hard to maintain. 

- similar function is placed in the `mirror node` client. Maybe we could use the pattern shown above and extract them into something coherent -> `TransactionId` or whatelse in a separate package